conn.go

```diff
@@ -97,7 +97,14 @@ type ConnConfig struct {
 
 // ReadBatchConfig is a configuration object used for reading batches of messages.
 type ReadBatchConfig struct {
+	// MinBytes indicates to the broker the minimum size a batch of messages must
+	// be when consuming. Setting a high minimum on a low-volume topic can result
+	// in messages appearing delayed if they never reach the minimum.
```

Comment:
This is a difficult concept to express clearly.  My attempt would be something like:

> MinBytes indicates to the broker the minimum batch size that the consumer will accept.  Setting a high minimum when consuming from a low volume topic may result in delayed delivery because the broker cannot return any messages until it has accumulated enough to satisfy the request.

Though I'd massage that further. 😁